### PR TITLE
updated tokenomics and added reward_mechanism_v2

### PR DESCRIPTION
**Pull request by ipanagiv**

### CODE REVIEW

File: docs/project/tokenomics.mdx [hunk 1]
```diff
@@ -1,80 +1,135 @@
...
+will be used to reward the company and the investors for their contributions and work.
+
+#### 10% - Ecosystem Development:
+tokens will be reserved for ecosystem development and the formation of the DAO.
+
+#### 5% - Liquidity Bootstrap:
+will be used to bootstrap and support $WXM’s liquidity.
+
+#### 3% Airdrop" to beta participants
+3 million tokens will be used to reward stations that contributed data to the Network during the beta phase. This is not a marketing activity intended to create buzz and awareness, but a reward mechanism for the stations that participated and formed the Network before a reward mechanism was in place.
```
Comment (PavlosTze) [inline docs/project/tokenomics.mdx:61]: Please after the `:` start the next sentence in the new line with an upper-case letter. Also the `:` is missing on the `3% Airdrop" to beta participants`

File: docs/project/tokenomics.mdx [hunk 2]
```diff
@@ -1,80 +1,135 @@
...
+As more logic is transferred into smart contracts, a need for grouping of functionality and modularity emerges. That is why multiple smart contracts are now created, each with its own logic and design patterns:
+
+- WeatherXM
+- WeatherXMData
+- WeatherStationXM
+- RewardPool
+- BurnPool
+- PriceFeedConsumer
+
+Diagrams illustrating the interaction among the smart contracts, storage slots, classes, etc can be found <a href="https://github.com/WeatherXM/smart-contracts/tree/develop/docs/diagrams">diagrams</a>.
```
Comment (PavlosTze) [inline docs/project/tokenomics.mdx:97]: Replace `diagrams` hyperlinked with `here` or added parentheses to it?

File: docs/project/tokenomics.mdx [hunk 3]
```diff
@@ -1,80 +1,135 @@
...
+
+
+## Assumptions
+- Different Admins can be assigned to each contract after deployment to the network by transferring ownership.
+
+- Admin has a lot of power, and is assumed to be the right, fair and just person/party. It is highly advised to have a multisig as admin, rather than just an EOA.
```
Comment (PavlosTze) [inline docs/project/tokenomics.mdx:103]: This needs better wording IMO, doesn't make much sense and what does EOA means?

File: docs/project/tokenomics.mdx [hunk 4]
```diff
@@ -1,80 +1,135 @@
...
+
+- Company and investors tokens are allocated everyday to a target EoA which is set by the RewardPool Admin. The Distributor, is responsible for initiating the transfer of company and investors tokens daily to the target EoA.
```
Comment (PavlosTze) [inline docs/project/tokenomics.mdx:105]: EoA ? Let's explain this please.

File: docs/project/reward-algorithm_v2.mdx [hunk 5]
```diff
@@ -0,0 +1,52 @@
+---
+id: reward-algorithm_v2
+title: Reward Algorithm_v2
+sidebar_label: Reward Algorithm_v2
```
Comment (PavlosTze) [inline docs/project/reward-algorithm_v2.mdx:4]: `Reward Algorithm_v2` doesn't look beautiful when it comes to people reading this. Let's rename this to `Reward Algorithm V2` please and remove the underscore and capitalize the `v`.

File: docs/project/reward-algorithm_v2.mdx [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+The rewards allocated for a cell will be distributed to the first N devices with best station reward score, where N=capacity of cell. In case two devices share the same score then they will be sorted based on seniority and the more senior one will get the rewards.
+
+ 
+
+
+:::note For example:
+If the capacity of a cell is 5 but has 6 stations and the Max Cell Reward is 20 WXM then the first 5 devices based on their performance in the previous day `station_reward_score` will be rewarded with a `station_reward_score`*`20/5`. This would have also been the case, if the specific cell had fewer than 5 stations.
+:::
+To read more please see our <a href="https://docs.google.com/document/d/1_z-sdCoSpm7H5kH0Ko8grUn7IatmAObSdkQULr21YGc/edit#heading=h.jgrgzl8vvkoy">whitepaper</a>
```
Comment (PavlosTze) [inline docs/project/reward-algorithm_v2.mdx:21]: We should **not** use the google docs link at any cases, as that might change in the future. Please let's use the static one we have control of:
[https://weatherxm.com/whitepaper](https://weatherxm.com/whitepaper)

File: docs/project/reward-algorithm_v2.mdx [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+## Claiming Rewards
+All station owners can withdraw their rewards at any time without time or other limit. Every owner initiates the process of withdrawing and signs the withdrawal transaction with his private key at the frontend (clients) through Metamask or another wallet, or programmatically by interacting directly with the smart contract. Therefore, a station owner can withdraw only his rewards. During withdrawing, the required parameters by the smart contract are the latest proof, his cumulative rewards, the latest cycle and finally the amount to claim which can be smaller or equal to the total amount of cumulative rewards. The cycle has been introduced in order to track the minting process. Every time a mint is completed, the cycle is increased and then the cycle is used as accessor in the root hashes mapping in the RewardPool.
+ 
+:::note
+ more on  <a href="https://github.com/WeatherXM/smart-contracts/blob/develop/docs/merkle_tree.md">reward claiming</a>.
```
Comment (PavlosTze) [inline docs/project/reward-algorithm_v2.mdx:51]: more -> More

**Review by PavlosTze — CHANGES_REQUESTED** [review]
@ipanagiv Some minor stuff here and there need fixing please :) Good job in general!

File: docs/project/tokenomics.mdx (hunk 2 above)
Comment (ipanagiv) [reply docs/project/tokenomics.mdx]: done

File: docs/project/tokenomics.mdx (hunk 3 above)
Comment (ipanagiv) [reply docs/project/tokenomics.mdx:103]: section removed

File: docs/project/reward-algorithm_v2.mdx (hunk 5 above)
Comment (ipanagiv) [reply docs/project/reward-algorithm_v2.mdx]: done!

File: docs/project/reward-algorithm_v2.mdx (hunk 6 above)
Comment (ipanagiv) [reply docs/project/reward-algorithm_v2.mdx]: you are correct. I only linked to the google doc, because it could go straight to the specific section.fixed

File: docs/project/reward-algorithm_v2.mdx (hunk 7 above)
Comment (ipanagiv) [reply docs/project/reward-algorithm_v2.mdx]: done

File: docs/project/reward-algorithm V2.mdx [hunk 8]
```diff
@@ -1,8 +1,8 @@
 ---
-id: reward-algorithm_v2
-title: Reward Algorithm_v2
-sidebar_label: Reward Algorithm_v2
-slug: /reward-algorithm_v2
+id: reward-algorithm V2
+title: Reward Algorithm V2
+sidebar_label: Reward Algorithm V2
+slug: /reward-algorithm V2
```
Comment (PavlosTze) [inline docs/project/reward-algorithm V2.mdx:5]: @ipanagiv Please keep "Reward Algorithm V2" only on `title` and `sidebar_label` and "reward-algorithm-v2" on `id` and `slug`.

It's better not to use spaces on `id` and on the `slug`.

File: sidebars.js [hunk 9]
```diff
@@ -8,7 +8,7 @@ module.exports = {
 				"project/project",
 				"project/tokenomics",
 				"project/reward-algorithm",
-				"project/reward-algorithm_v2",
+				"project/reward-algorithm V2",
```
Comment (PavlosTze) [inline sidebars.js:11]: So in addition to my above comment this will become reward-algorithm-v2

**Review by PavlosTze — CHANGES_REQUESTED** [review]
Only 1 very minor thing left and it's ok by me :)

File: docs/project/reward-algorithm V2.mdx (hunk 8 above)
Comment (ipanagiv) [reply docs/project/reward-algorithm V2.mdx]: Thanks

**Review by PavlosTze — APPROVED** [review]
Approved by me, good job @ipanagiv . @epapangelis up to you now Vag :)

File: docs/project/reward-algorithm V2.mdx [hunk 10]
```diff
@@ -31,42 +33,45 @@ The daily distribution of rewards will amount to 14246. After the rewarding algo
...
 
 :::note For example:
-If the capacity of a cell is 5 but has 6 stations and the Max Cell Reward is 20 WXM then the first 5 devices based on their performance in the previous day `station_reward_score` will be rewarded with a `station_reward_score`*`20/5`. This would have also been the case, if the specific cell had fewer than 5 stations.
+If the capacity of a cell is 5 but has 6 stations and *reward_score=10* then the first 5 devices based on their performance in the previous day will be awarded *station_reward_score x 10*. If there are two stations in the 5th place, with exactly the same *station_reward_score*, then the more senior of the two will get the rewards.
```
Comment (chatper) [inline docs/project/reward-algorithm V2.mdx:55]: I would rephrase that as
```
If the capacity of a cell is 5, but has 6 stations with a random *reward_score*, then only the first 5 devices with the highest *reward_score* will be rewarded. If there are two stations in the 5th place, with exactly the same *station_reward_score*, then the more senior of the two will get the rewards
```

File: docs/project/reward-algorithm V2.mdx [hunk 11]
```diff
@@ -31,42 +33,45 @@ The daily distribution of rewards will amount to 14246. After the rewarding algo
...
 ### Fair
 All rewards will be distributed according to the rules and methods defined in this document and any changes will be publicly discussed before decisions are made. 
 ### Verifiable
-All individual scores that lead to the final reward, if reproduced, will produce the same result. For as long as the Company is responsible for calculating these scores and distributing the rewards, all required software, configuration and other technical details required by third parties to reproduce and validate them, will be made publicly available. All relevant software created by the Company for this purpose will be developed and distributed under an open source licence.
+All individual scores that lead to the final reward, if reproduced, will produce the same result. For as long as WeatherXM AG is responsible to calculate these scores and distribute the rewards, all required software, configuration and other technical details needed by a a third party to reproduce and validate them, will be made publicly available. All relevant software created by WeatherXM AG for this purpose will be developed and distributed under an open source licence.
```
Comment (chatper) [inline docs/project/reward-algorithm V2.mdx:73]: There's a typo here: `... needed by a a third party...`

**Review by tmavroeid — APPROVED** [review]
